main.go

```diff
@@ -1,22 +1,18 @@
 package main
 
 import (
+	"context"
+	"fmt"
```

Comment:
This is mainly updating main, server and service.go to use latest microkit. This is so I could introduce helmclient for installing / upgrading tiller.